openssl/src/ssl/mod.rs

```diff
@@ -3484,6 +3484,63 @@ impl SslRef {
             }
         }
     }
+
+    /// this returns the hash, e.g. SHA1, SHA512
```

Comment:
What hash?

Reply:
I just updated the comment to more explicitly refer to the signature scheme. I find the `signature_nid` and `signature_type_nid` naming to be really confusing because it's not totally obvious which one returns the digest and which returns the signature algorithm, so I though it would be useful to include an example.